### ISSUE TITLE
CSV Import Encryption Key

### ISSUE THREAD

**Issue by afnan-ahmad** [realm/realm-cocoa-converter#28]
I imported a CSV using sample code given in the **Readme.md** file. When I try to open generated file using **RealmBrowser** it is asking for 128-character encryption. Any idea what it might be; how to avoid it or set our own?

**Comment by stel** [comment]
What version of RealmConverter and Realm Browser do you use?

**Comment by stel** [comment]
@afnan-ahmad did you solve this problem? Can we close this issue?

**Comment by afnan-ahmad** [comment]
@stel yes, I solved this issue. The problem was in my CSV file format that was causing this issue.

**Comment by stel** [comment]
Ok, thanks! Was this issue related to RealmConverter? If so, could you please share this CSV file and the info how to reproduce it?

**Comment by afnan-ahmad** [comment]
I was using DB Browser for SQLite to export tables data to CSV so I can use them to import into realm db. By defaults that was exporting as `Unix: LF (\n)` as new line character, which was resulting this issue. Changing it to `Windows: CR+LF (\r\n)` format worked with it.
